### PR TITLE
Translation Delivery

### PR DESCRIPTION
**Pull request by service-tip-git**
Translation Delivery

### CODE REVIEW

**Review by marcauberer — DISMISSED** [review]

**Review by MikeMcC399 — COMMENTED** [review]
I commented on the English translation. I am unsure about whether it accurately reflects the intention of the German text "Lassen Sie sich testen, z.B. mit einem Antigen-Schnelltest, oder testen Sie sich selbst, am besten mehrfach im Abstand von zwei Tagen."

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -272,7 +272,7 @@
     <!-- XTXT: find details about exposures in the contact journal - increased risk -->
     <string name="risk_details_find_details_in_journal_increased_risk">"You will find more information about your exposures in your contact journal.\n\nThere is an increased risk that you have been infected. If you have been infected, you can infect other people, even if you have been vaccinated or have recovered and even if you don’t have any symptoms."</string>
     <!-- XMSG: risk details - link to faq, something like a bullet point -->
-    <string name="risk_details_increased_risk_faq_link_text">"Please get tested, even if you don’t have any symptoms. If you get a warning of increased risk (“red tile”), you are entitled to a test free of charge. It does not necessarily have to be a PCR test."</string>
+    <string name="risk_details_increased_risk_faq_link_text">"Please get tested, for example, with a rapid antigen test, or use self-tests - preferably several over two days. If a test is positive, take a PCR test to confirm the result."</string>
```
[the remark this answers is not in the document]
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values/strings.xml:275]: Hi @MikeMcC399, thanks for the observation. I'll let the team know about this.

[the remark this answers is not in the document]
Comment (MikeMcC399) [reply Corona-Warn-App/src/main/res/values/strings.xml:275]: @SamuraiKek 

Many thanks! The English and German texts (and presumably other languages too) need to be clear for repeat testing:
- what is the recommended interval between tests?
- when can the user stop testing?

**Comment by service-tip-git** [comment]
This pull request is for translation delivery. Please review and approve them.

**Review by SamuraiKek — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply Corona-Warn-App/src/main/res/values/strings.xml]: @SamuraiKek 

My comment about the English text not matching the German text is resolved in today's commit: https://github.com/corona-warn-app/cwa-app-android/pull/5317/commits/5434d2d9671f21f70330e85f5d2d565818cef088.

[the remark this answers is not in the document]
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Thanks @MikeMcC399, I'll mark this conversation resolved. The other languages have recieved the same treatment. I can confirm with Romanian at least 😄

**Pull request merged**